htmd/simlist.py

```diff
@@ -12,6 +12,7 @@
 from htmd.molecule.molecule import Molecule
 from joblib import Parallel, delayed
 from htmd.progress.progress import ProgressBar
+import mdtraj as md
 import logging
```

Comment:
Yes, sorry about the messy code, I will clean up what I have written as I introduced a lot of redundancy! I did manage to get it to work with pmemd now, at last!